x-pack/plugin/core/src/test/java/org/elasticsearch/protocol/xpack/XPackInfoResponseTests.java

```diff
@@ -21,8 +21,10 @@
 import java.util.function.Function;
 import java.util.function.Predicate;
 import java.io.IOException;
+import java.util.stream.Collectors;
 
-public class XPackInfoResponseTests extends AbstractStreamableXContentTestCase<XPackInfoResponse> {
+public class XPackInfoResponseTests extends AbstractHLRCStreamableXContentTestCase<XPackInfoResponse,
+    org.elasticsearch.client.xpack.XPackInfoResponse> {
```

Comment:
Could you indent this one more time?  It makes it hard to read as is.

Reply:
done in b5372c4